src/app/features/bitcoin-choose-fee/bitcoin-choose-fee-loader.tsx

```diff
@@ -0,0 +1,19 @@
+import { Flex } from '@stacks/ui';
+
+import { LoadingSpinner } from '@app/components/loading-spinner';
+
+interface BitcoinChooseFeeLoaderProps {
+  children: JSX.Element;
+  isLoading: boolean;
+}
+
+export function BitcoinChooseFeeLoader({ children, isLoading }: BitcoinChooseFeeLoaderProps) {
+  if (isLoading) {
+    return (
+      <Flex py="108px" justifyContent="center" alignItems="center" width="100%">
+        <LoadingSpinner />
+      </Flex>
+    );
+  }
+  return children;
+}
```

Comment:
This component doesn't actually do any loading, so as not to confuse with other "loader pattern", suggest rename. Separate loading component is a nice idea though, alternatively could live in layout component?

Reply:
you mean put LoadingSpinner in Layout component?